Runner.php

```diff
@@ -15,34 +16,28 @@ public function __construct($config, $server)
 
     public function queueDirectory($dir, $prefix = '')
     {
-        if(isset($this->config['include_paths'])) {
-            $this->queueWithIncludePaths();
-        } else {
-            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
```

Comment:
If the answer comes back yes, can you split the PR into 2: one that ditches exclude-paths support and another with your `./` bugfix?